File: Medium-Questions/15-Triangle-Area/C#/fifteen.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace fifteen
{
    class Program
    {
        static void Main(string[] args)
        {

            // get input and convert it
            Console.Write("Enter width: ");
            string temp = Console.ReadLine();
            int width = Int32.Parse(temp);

            // get input and convert it
            Console.Write("Enter heigth: ");
            string temp1 = Console.ReadLine();
            int heigth = Int32.Parse(temp);

            // do calculation
            double answer = (width * heigth) * 0.5;
            Console.WriteLine("Area is {0}",answer);


            Console.ReadKey();
        }
    }
}

File: Medium-Questions/15-Triangle-Area/C/fifteen.c
#include <stdio.h>



int main(void)
{
    int width;
    int height;

    printf("Enter width of triangle:");
    scanf("%d", &width);

    printf("Enter height of triangle:");
    scanf("%d", &height);

    float answer = (width * height) * 0.5;


    printf("Area is %f", answer);
    return 0;

}